Sming/SmingCore/Network/WebsocketClient.cpp

```diff
@@ -99,10 +100,11 @@ void WebsocketClient::onError(err_t err)
 
 bool WebsocketClient::_verifyKey(char* buf, int size)
 {
-	String dd = String(buf);
-	uint8_t s = dd.indexOf("Sec-WebSocket-Accept: ");
+	String keyHeader("Sec-WebSocket-Accept: ");
+	String dd(buf);
```

Comment:
This code assumes that the header will be in one tcp packet, which is wrong. It may cause broken validation.

```
switch (_mode)
		{
		case wsMode::Connecting:
			if (_verifyKey((char*)data, size) == true)
```